src/main/scala/com/gu/sfl/exception/SaveForLaterError.scala

```diff
@@ -1,13 +1,15 @@
 package com.gu.sfl.exception
 
 sealed trait SaveForLaterError {
-  self: Throwable =>
+  def message: String
 }
 
-case class MissingAccessTokenException(message: String) extends Exception(message) with SaveForLaterError
-case class UserNotFoundException(message: String) extends Exception(message) with SaveForLaterError
-case class IdentityServiceException(message: String) extends Exception(message) with SaveForLaterError
-case class SavedArticleMergeError(message: String) extends  Exception(message) with SaveForLaterError
-case class MaxSavedArticleTransgressionError(message: String) extends Exception(message) with SaveForLaterError
-case class IdentityApiRequestError(message: String) extends Exception(message) with SaveForLaterError
-case class RetrieveSavedArticlesError(message: String) extends Exception(message) with SaveForLaterError
+case class MissingAccessTokenException(message: String) extends SaveForLaterError
```

Comment:
I think you just need to rename the errors in this file so they aren't named Exception

Reply:
Yes i totally meant to do that. Thanks. :(